contracts/extensions/TokenSupplier.sol

```diff
@@ -111,15 +96,8 @@ contract TokenSupplier is ColonyExtension {
 
   /// @notice Update the tokenSupplyCeiling, cannot set below current tokenSupply
```

Comment:
Has preventing it being set below tokenSupply been lost somewhere? Does this matter?

Reply:
I don't think it matters, really. Tokens can be minted at any time for any reason, so the totalSupply may always become spontaneously larger than the ceiling. But for sanity's sake maybe best to prevent users from doing it here.